x/tss/module.go

```diff
@@ -126,6 +131,8 @@ func (AppModule) QuerierRoute() string {
 // RegisterServices registers a GRPC query service to respond to the
 // module-specific GRPC queries.
 func (am AppModule) RegisterServices(cfg module.Configurator) {
```

Comment:
Also need to update `RegisterRESTRoutes` and `RegisterGRPCGatewayRoutes`

Reply:
`RegisterGRPCGatewayRoutes ` is already updated.